AdyenCard/Form/Installments/FormCardInstallmentsItemView.swift

```diff
@@ -0,0 +1,22 @@
+//
+// Copyright (c) 2021 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Adyen
+
+internal typealias InstallmentPickerElement = BasePickerElement<InstallmentElement>
+
+internal final class FormCardInstallmentsItemView: BaseFormPickerItemView<InstallmentElement> {
+    override internal func initialize() {
```

Comment:
You mean change the code of the super class? I just followed other examples of sibling classes.

Reply:
wondering if its necessary?: or if it can be much better to add this to the super class, we shouldn't be liberal when it comes to opening stuff for overriding, its dangerous and limit our options for refactoring.